packages/hierarchies-react/src/presentation-hierarchies-react/stratakit/TreeNodeRenderer.tsx

```diff
@@ -82,23 +101,23 @@ export const StrataKitTreeNodeRenderer: FC<PropsWithRef<TreeNodeRendererProps &
     const label = useMemo(() => (getLabel ? getLabel(node) : node.label), [getLabel, node]);
     const description = useMemo(() => (getSublabel ? getSublabel(node) : undefined), [getSublabel, node]);
     const decorations = useMemo(() => getDecorations?.(node), [getDecorations, node]);
-    const actions = useMemo(
-      () => [
-        ...(node.error?.type === "Unknown"
-          ? [
-              <Tree.ItemAction
-                key="retry"
-                label={localizedStrings.retry}
-                onClick={() => reloadTree({ parentNodeId: node.id, state: "reset" })}
-                visible={true}
-                icon={refreshSvg}
-              />,
-            ]
-          : []),
-        ...(getActions ? getActions(node) : []),
-      ],
-      [getActions, node, localizedStrings, reloadTree],
-    );
+    const inlineActions = useMemo(() => {
+      if (node.error !== undefined && node.error.type === "Unknown") {
+        return [
```

Comment:
Previous implementation was rendering `Retry` & other actions. Now we render only `Retry` action. Is this intended?

Reply:
Yes, from `stratakit` example of how to handle errors, we should only render actions that would solve the error as other actions would not provide any value and could have unexpected behavior because of the error.